tests/uefistored/conftest.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+from contextlib import contextmanager
+from collections import namedtuple
+import ntpath
+import shlex
+
+from lib.common import is_uuid, wait_for
+from lib.efi import (
+    Certs,
+    EFIAuth,
+    EFI_AT_ATTRS,
+    EFI_GUIDS,
+)
+
+VM_SECURE_BOOT_FAILED = 'VM_SECURE_BOOT_FAILED'
+
+@pytest.fixture
+def sb_vm_unsigned(running_vm):
+    vm = running_vm
+
+    print('> rm uefistored guest certs')
```

Comment:
If it is a requirement that hosts and VMs are already in a clean state, lines 21-26 are not needed because the only purpose of these lines is to guarantee that previous state won't interfere with the current test.

Reply:
Clean-ish, actually. If just booting the VM previous tests may add `VM_SECURE_BOOT_FAILED` messages for that VM, then it's probably better to keep the clean-up and just add a comment stating why it's done.